src/AssemblerHttpServer-Alignments.cpp

```diff
@@ -1590,7 +1596,7 @@ void Assembler::exploreAlignmentGraph(
 
     // Write the form.
     html <<
-        "<h3>Display a local subgraph of the <a href='docs/ReadGraph.html'>global alignment graph</a></h3>"
+        "<h3>Display a local subgraph of the global alignment graph</h3>"
```

Comment:
In an ideal world, it would be useful to have links to the documentation from the http server. We could even have documentation specific to the http server that explains how to use a page, for example. There are two scenarios where this currently works (or, at least, should work): when the server runs from the `install` directory (possibly obtained from a `tar` artifact), and when the server runs from the `AppImage`. We are currently not distributing the `AppImage`, but we could consider switching to that, instead of the static executable.

We can think about this in the future, but I agree on removing this link for now.

Reply:
`AppImage` has come up twice in our conversations. I need to look into what that is :)

Reply:
You will find lots of documentation and information on it. An AppImage contains the executable plus all the dynamic libraries it requires plus all the support files you want (e. g. for us `docs`, `conf`) and stores them in compressed form in a single file. At execution time, all of those files are mounted under the covers in a filesystem (without decompressing them) and made available to the executable. For example, the way it is built now for Shasta, the executable knows that the `docs` directory is always available at `../docs` relative to the location of the executable, just like when the executable runs form the `shasta-install/bin` directory.

The AppImage is supposed to be portable across Linux systems. However there is a guideline that says you should build it on the oldest system you want to support, which could be problematic for us. I am not sure it this is a strict requirement though.